Vagrantfile

```diff
@@ -0,0 +1,131 @@
+# -*- mode: ruby -*-
```

Comment:
I have discussed an approach to removing these altogether with Avtar, and have a plan to fix all projects in the stack.  See [GPII-1985](https://issues.gpii.net/browse/GPII-1985)